src/supermarket/app/views/application/_search.html.erb

```diff
@@ -57,7 +57,7 @@
     </div>
 
     <div class= "advanced_textfield">
-      <%= search_field_tag 'platforms[]', nil, placeholder: "You can write platform name here if it's not on the list", class: 'platform_search_textfield' %>
+      <%= search_field_tag 'platforms[]', nil, placeholder: "You can write in a platform name here if it's not listed above" class: 'platform_search_textfield' %>
```

Comment:
Your edit ate a comma separating the string value of `:placeholder` and the next key `:class`.